src/Components/Calculator/Calculator.js

```diff
@@ -2,6 +2,7 @@ import React, { useState } from 'react';
 import './Calculator.css';
 import PropTypes from 'prop-types';
 import calculate from '../Logic/calculate';
+import Quotes from '../Quotes/quotes';
 
 const Calculator = () => {
   const [total, setTotal] = useState(null);
```

Comment:
- [ ] Nice work you did 👏 . However, the instructions was to Import the new component into your `main` component (next to the Calculator)
- It should then be on the same page as the Calculator

## Sample:
```
import Calculator from "./components/Calculator"
import Quotes from "./components/Quotes"

function App() {
  return (
    <>
      {/* Rest of JSX */}
      <Calculator />
      <Quotes />
    </>
  )
}

export default App
```